### PR TITLE
Adding GoogleNlp Client:

### PR DESCRIPTION
**Pull request by AMCR**
#### Label: ####
_feature_
#### What's this PR do?
 - Akka stream implementation of Google NLP Client
 - Adding Tests
#### Where should the reviewer start?
akka-google-nlp
#### What tests were added?
Yes
#### Any background context you want to provide?
GoogleNLP Api 1.2 Beta
#### Are there other tickets linked to this?
https://jira.aws.telegraph.co.uk/browse/NLP-128 
https://jira.aws.telegraph.co.uk/browse/NLP-127
#### Definition of Done:
- [x] Is there appropriate test coverage?
- [ ] Does the confluence need to be updated?
- [ ] Does this add new dependencies? If so, have the healthchecks been updated?
- [ ] Any Deployment changes we need to be aware of? New infrastructure? Changes to existing components?
- [x] Is there appropriate logging included?
- [ ] Does it imply any kind of data migration? If so what are the steps for that?
- [ ] Is the Documentation been updated, if needed?

### CODE REVIEW

File: akka-google-nlp/src/main/scala/uk/co/telegraph/google/nlp/NlpSettings.scala [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    extractEntities          -> NlpAnalyseEnum.Entities,
+    extractDocumentSentiment -> NlpAnalyseEnum.DocumentSentiment,
+    extractEntitySentiment   -> NlpAnalyseEnum.EntitySentiment,
+    classifyText             -> NlpAnalyseEnum.ClassifyText
+  ).collect{ case (true, value) => value }
+}
+
+
+object NlpSettings {
+  private val prefix = "google.api.nlp"
```
Comment (carlossegura) [inline akka-google-nlp/src/main/scala/uk/co/telegraph/google/nlp/NlpSettings.scala]: would it be worth it getting this from a property file?

File: src/main/scala/uk/co/telegraph/ucm/enricher/routes/SampleEndpoints.scala [hunk 2]
```diff
@@ -1,12 +1,12 @@
-package uk.co.telegraph.ucmcontentenricher.routes
+package uk.co.telegraph.ucm.enricher.routes
```
Comment (carlossegura) [inline src/main/scala/uk/co/telegraph/ucm/enricher/routes/SampleEndpoints.scala]: is this file still needed?

File: akka-google-nlp/src/test/scala/uk/co/telegraph/google/nlp/model/response/NlpResultSpecs.scala [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package uk.co.telegraph.google.nlp.model.response
+
+import org.scalatest.{FreeSpec, Matchers}
+
+class NlpResultSpecs
```
Comment (Painyjames) [inline akka-google-nlp/src/test/scala/uk/co/telegraph/google/nlp/model/response/NlpResultSpecs.scala]: empty test needed?

File: akka-google-nlp/src/main/scala/uk/co/telegraph/google/nlp/NlpSettings.scala (hunk 1 above)
Comment (AMCR) [reply akka-google-nlp/src/main/scala/uk/co/telegraph/google/nlp/NlpSettings.scala]: the idea is to have a default prefix. This is a common strategy when you want to have default properties. Imagine this scenario: your configuration needs to have 3 properties. by default they are located under:
```
#reference.conf
google.api.nlp: 
   prop1: value1
   prop2: value2
   prop3: value3
```
and
```
#application.xxx.conf
app.google-nlp: 
   prop1: value1
   prop2: value2
```

Because we always load the reference.conf and since we are using the *inner withFallback root.getConfig(prefix)* it means that if the configuration does not exist in your *application.xxx.conf* it will load it from the reference

File: src/main/scala/uk/co/telegraph/ucm/enricher/routes/SampleEndpoints.scala (hunk 2 above)
Comment (AMCR) [reply src/main/scala/uk/co/telegraph/ucm/enricher/routes/SampleEndpoints.scala]: The sample endpoints no. We need to rename it and add the proper routes. it will be added in the next ticket.

File: akka-google-nlp/src/test/scala/uk/co/telegraph/google/nlp/model/response/NlpResultSpecs.scala (hunk 3 above)
Comment (AMCR) [reply akka-google-nlp/src/test/scala/uk/co/telegraph/google/nlp/model/response/NlpResultSpecs.scala]: It is no longer empty :)
